tesseract/tesseract_planning/tesseract_process_managers/include/tesseract_process_managers/process_managers/simple_process_manager.h

```diff
@@ -1,92 +0,0 @@
-/**
- * @file simple_process_manager.h
- * @brief Plans simple paths aka. a single composite containing no composite instruction.
- *
- * @author Matthew Powelson
- * @date July 15. 2020
- * @version TODO
- * @bug No known bugs
- *
- * @copyright Copyright (c) 2020, Southwest Research Institute
- *
- * @par License
- * Software License Agreement (Apache License)
- * @par
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- * http://www.apache.org/licenses/LICENSE-2.0
- * @par
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-#ifndef TESSERACT_PROCESS_MANAGERS_SIMPLE_PROCESS_MANAGER_H
-#define TESSERACT_PROCESS_MANAGERS_SIMPLE_PROCESS_MANAGER_H
-
-#include <tesseract_common/macros.h>
-TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
-#include <functional>
-#include <vector>
-#include <thread>
-#include <taskflow/taskflow.hpp>
-TESSERACT_COMMON_IGNORE_WARNINGS_POP
-
-#include <tesseract_process_managers/core/process_manager.h>
-#include <tesseract_process_managers/core/taskflow_generator.h>
-
-namespace tesseract_planning
-{
-/**
- * @brief This class provides a process manager for a simple process.
- *
- * Given a ProcessInput in the correct format, it handles the creation of the process dependencies and uses Taskflow to
- * execute them efficiently in a parallel based on those dependencies.
- *
- * The required format is below.
- *
- * Composite
- * {
- *   ... Must not contain any child composite instructions
- * }
- */
-class SimpleProcessManager : public ProcessManager
```

Comment:
I guess you're thinking we are ready to remove this? I have not used the planning server yet. That provides the same functionality?

Reply:
I thought it would be good to remove before merging into master, since the planner server now provides the same functionality. Mainly to prevent other from using it. Do you want to keep it?

Reply:
You can remove it.

Reply:
The only thing I have noticed with the planning server is that having a single executor it does not release memory for some cases. I have created a test case and posted an issues [here](https://github.com/taskflow/taskflow/issues/264).

Reply:
Interesting. That does seem like a pretty big deal actually - though probably a taskflow issue. So if we call make_shared in the taskflow anywhere that is essentially a memory leak?

Reply:
Actually, that is really bizarre. Have you tried running valgrind on it?

Reply:
> Interesting. That does seem like a pretty big deal actually - though probably a taskflow issue. So if we call make_shared in the task anywhere that is essentially a memory leak?

It does not appear to be a memory leak for this case, but something related libc based on his reply. I have asked  for more details.

Reply:
I have found that if I submit another request it does not appear to increase it memory but use what it has already taken ownership of. I do have concerns that this may starve other process but may not be the case per the authors comments related to modern OS handling this.